compiler/modules/nback/init.bal

```diff
@@ -152,7 +160,9 @@ function addInherentTypeDefn(InitModuleContext cx, string symbol, t:SemType semT
 }
 
 function addArrayInherentTypeDefn(InitModuleContext cx, string symbol, int tid, t:UniformTypeBitSet bitSet) returns [llvm:StructType, llvm:ConstPointerValue] {
-    llvm:ConstValue initValue = cx.llContext.constStruct([llvm:constInt(LLVM_TID, tid), getListGetFunc(cx), getListSetFunc(cx), llvm:constInt(LLVM_BITSET, bitSet)]);
+    t:UniformTypeCode? utCode = t:uniformTypeCode(bitSet);
```

Comment:
addArrayInherentTypeDefn only gets called once for each InherentType.  So it's only going to get called at most once for int array and at most once for float array.  So this call can unconditionally declare the get/setFuncs.  You don't need all these tables.  The int/float and generic cases are not parallel: the generic case will get called for multiple inherent types; the specialized case for one each.